pkgs/development/libraries/snappy/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, cmake }:
+{ stdenv, fetchFromGitHub, cmake, sharedLibrary ? false }:
```

Comment:
I would rather build shared libraries by default. Which is what we prefer in nixpkgs.

Reply:
This variable does not have an effect on this derivation, was this added by mistake?